frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_3__Gene Expression Data Processing.mdx

```diff
@@ -75,58 +75,31 @@ Only cells from sequencing assays that measure gene expression and don't require
 
 ## Gene-length Pre-normalization
 
-**WARNING** - The following process has not been implemented as the feature **currently only includes data that does not require gene-length normalization.**
```

Comment:
I believe we now do this consistently - removing the warning. Lmk if this is not the case, @atarashansky .